### PR TITLE
Tweaks from Sprint Review

### PR DESCRIPTION
**Pull request by jjbennett**
# Critical Changes

# Changes

# Issues Closed

# New Metadata

# Deleted Metadata

# Definition of Done
  Refer to [Asteroids DoD document](https://salesforce.quip.com/iq2mAy4i62oM) to see any additional details for the items below
~~Any net new LWC work has Sa11y tests & 50% or above lines JEST test coverage~~
~~CRUD/FLS is enforced in Apex~~
~~Permission sets are updated to account for CRUD|FLS|Tab|Classes~~
~~Field sets are updated to account for new fields~~
~~UX approval or UX not necessary~~
- [ ] Link the pull request and work item by PR comment and Chatter post respectively, e.g. GUS: W-0000000: Work Name (https://github.com/SalesforceFoundation/PMM/pull/303)
- [ ] All **acceptance criteria** have been met
    - [ ] Developer
    - [ ] Code Reviewer
    - [ ] QA
~~PR contains draft release notes~~
- [ ] QE story level testing completed

### CODE REVIEW

**Comment by jjbennett** [comment]
[W-8595069](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07B0000008kg4fIAA/view)

File: force-app/main/default/labels/CustomLabels.labels-meta.xml [hunk 1]
```diff
@@ -680,7 +680,7 @@
         <language>en_US</language>
         <protected>true</protected>
         <shortDescription>Warning when list view limit is reached.</shortDescription>
-        <value>There are over 2,000 records. Some records may not be shown.</value>
+        <value>This List View has over 2,000 records. Some records may not be shown.</value>
```
Comment (bethbrains) [inline force-app/main/default/labels/CustomLabels.labels-meta.xml:683]: @lbeller  - `list view` or `List View`?

**Review by bethbrains — APPROVED** [review]
One quick Q for @lbeller  but otherwise looks great. You are so thorough, @jjbennett !!

File: force-app/main/default/labels/CustomLabels.labels-meta.xml (hunk 1 above)
Comment (lbeller) [reply force-app/main/default/labels/CustomLabels.labels-meta.xml:683]: list view

**Review by lbeller — CHANGES_REQUESTED** [review]
Change requested: change List View to list view

**Comment by jjbennett** [comment]
Thanks for catching that @bethbrains and for the quick feedback @lbeller - the capitalization has been fixed.

**Review by lbeller — APPROVED** [review]

**Pull request merged**